doc/manuals/orion-api.md

```diff
@@ -1,127 +1,81 @@
-* Simple Query Language: Sub-key filtering (both attribute and metadata values, i.e. `q` and `mq`)
-* Notification metadata filtering
-* System/builtin metadata in notifications: `previousValue` and `actionType`
-
-## Copyright
-
-Copyright (c) 2011-2018 Telecom Italia, Telefónica I+D and NEC.
-
-## License
-
-This specification is licensed under the
-[FIWARE Open Specification License (implicit patent license)](https://forge.fiware.org/plugins/mediawiki/wiki/fiware/index.php/Implicit_Patents_License).
-
-## Conformance
-
-This specification describes the "full" compliance level.
-
-## Conventions
-
-NGSI version 2 uses camel case (camelCase) syntax for naming properties and related artifacts used
-by the API. When referring to URIs, as part of HATEOAS patterns, and to mark them appropriately,
-the suffix `_url` is added.
-
-## Reference Implementations
-
-* NGISv2 Context Brokers
-  * [Orion Context Broker](http://catalogue.fiware.org/enablers/publishsubscribe-context-broker-orion-context-broker) - [Implementation Notes](https://fiware-orion.readthedocs.io/en/master/user/ngsiv2_implementation_notes/index.html)
+<!-- TOC -->
+
+- [Specification](#specification)
+    - [Introduction](#introduction)
+    - [Terminology](#terminology)
+        - [Context data modelling and exchange](#context-data-modelling-and-exchange)
+            - [Context Entities](#context-entities)
+            - [Context Attributes](#context-attributes)
+            - [Context Metadata](#context-metadata)
+    - [MIME Types](#mime-types)
+    - [JSON Entity Representation](#json-entity-representation)
+    - [JSON Attribute Representation](#json-attribute-representation)
+    - [Simplified Entity Representation](#simplified-entity-representation)
+    - [Partial Representations](#partial-representations)
+    - [Special Attribute Types](#special-attribute-types)
+    - [Builtin Attributes](#builtin-attributes)
+    - [Special Metadata Types](#special-metadata-types)
+    - [Builtin Metadata](#builtin-metadata)
+    - [Field syntax restrictions](#field-syntax-restrictions)
+    - [Attribute names restrictions](#attribute-names-restrictions)
+    - [Metadata names restrictions](#metadata-names-restrictions)
+    - [Ordering Results](#ordering-results)
+    - [Error Responses](#error-responses)
+    - [Geospatial properties of entities](#geospatial-properties-of-entities)
+        - [Simple Location Format](#simple-location-format)
+        - [GeoJSON](#geojson)
+    - [Simple Query Language](#simple-query-language)
+    - [Geographical Queries](#geographical-queries)
+        - [Query Resolution](#query-resolution)
+    - [Filtering out attributes and metadata](#filtering-out-attributes-and-metadata)
+    - [Notification Messages](#notification-messages)
+    - [Custom Notifications](#custom-notifications)
+    - [Group API Entry Point](#group-api-entry-point)
+        - [Retrieve API Resources [GET /v2]](#retrieve-api-resources-get-v2)
```

Comment:
This is an API call (although pretty useless by the time being ;). It should be included in the "API Routes" section.

Reply:
Done in https://github.com/telefonicaid/fiware-orion/pull/4142/commits/cf9b206d5d224420155f1b6d08548a7ab48046f1